linux/before-nrtest.sh

```diff
@@ -1,13 +1,15 @@
 #! /bin/bash
-
 # 
 #  before-test.sh - Prepares Travis CI worker to run swmm regression tests
 #
 #  Date Created: 04/05/2018
 #
-#  Author:       Michael E. Tryby
+#  Authors:      Michael E. Tryby
 #                US EPA - ORD/NRMRL
 #
+#                Caleb A. Buahin
+#                Xylem Inc.
+#
 #  Arguments: 
 #    1 - relative path regression test file staging location 
```

Comment:
Take a look at the before-nrtest.cmd script. The argument was changed to RELEASE_TAG. This makes it really easy to grab whatever version of the benchmark you want for easy comparison.